openapi-integration-tests/src/test/java/io/ballerina/openapi/cmd/BallerinaToOpenAPICLINegativeTests.java

```diff
@@ -71,6 +71,34 @@ public void testForGivenBalServiceHasCompilationIssue() throws IOException, Inte
         assertOnErrorStream(process, out);
     }
 
+    @Test(description = "Test for given bal service file example value has json parser issues")
+    public void testForExamplesHasCompilerErrorse() throws IOException, InterruptedException {
+        String balFilePath = "examples/negative_test/service.bal";
+        List<String> buildArgs = new LinkedList<>();
+        buildArgs.add(0, "openapi");
+        buildArgs.add("-i");
+        buildArgs.add(balFilePath);
+        buildArgs.add("-o");
+        buildArgs.add(tmpDir.toString());
+
+        Process process = getProcess(buildArgs, TEST_RESOURCE);
+
+        String out = "openapi contract generation is skipped because of the following error(s) occurred " +
+                "during the code generation:\n" +
+                "WARNING [main.bal:(12:25,25:14)] Given example path `examples01.json` " +
+                "is not exist, therefore generated yaml does not contain example for `responseExample01` \n" +
+                "WARNING [main.bal:(12:25,25:14)] The given JSON value for example contains parsing issues, " +
```

Comment:
This location/message is not useful for finding which example caused this issue
Is it possible to narrow the location rather than pointing to the entire annotation?

Reply:
This entire location is due to the implementation earlier we consider all parts as one JSON file and read option. Therefore, wish to revisit this location and read the JSON value there once we merge all the example implementations.